docs/how-to-use-monty/getting-started/getting-started-on-windows-via-wsl.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting Started on Windows via WSL
+description: How to get the code running using Windows Subsystem for Linux and Visual Studio Code
+---
+> [!NOTE]
+> This guide is based off the main [Getting Started](getting-started) guide for macOS and Linux, which you may refer to as needed.
+
+> [!WARNING]
+> While the repository contains a `uv.lock` file, this is currently experimental and not supported. In the future this will change, but for now, avoid trying to use `uv` with this project.
+
+
+# 1. Install WSL and Ubuntu
+
+Follow the Microsoft guide to install Windows Subsystem for Linux: https://learn.microsoft.com/en-us/windows/wsl/install-manual
+
+It is important that you complete every step of their guide, otherwise you may encounter issues during the next steps below.
+
+Regarding the Linux distribution, [Ubuntu 24.04 LTS](https://apps.microsoft.com/detail/9nz3klhxdjp5) is recommended. Once installed, the Linux filesystem can directly be accessed via the "Linux" section in the Windows File Explorer, which should normally point to this path: `\\wsl.localhost\Ubuntu-24.04\`
+
+
+# 2. Install Miniconda
+
+Monty requires [Conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) to install its dependencies. For WSL, [Miniconda](https://www.anaconda.com/docs/getting-started/miniconda/install#linux-2) is recommended.
+
+To download and install Miniconda, enter the following commands in the Ubuntu terminal:
+
+```
+mkdir -p ~/miniconda3
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda3/miniconda.sh
+bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3
+rm ~/miniconda3/miniconda.sh
+source ~/miniconda3/bin/activate
+conda init --all
+```
+
+
```

Comment:
We'd prefer not to copy over instructions from the Conda website, so if they change things and this becomes out of date, we don't have to remember to come back and change it here.

```suggestion
```

Reply:
How about this:

> Monty requires [Conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) to install its dependencies. For WSL, [Miniconda](https://www.anaconda.com/docs/getting-started/miniconda/main) is recommended. To download and install Miniconda, follow this guide using the Ubuntu terminal: https://www.anaconda.com/docs/getting-started/miniconda/install#linux-2